Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+
+enum MenuOption
+{
+    Add = 1,
+    Subtract = 2,
+    Multiply = 3,
+    Divide = 4
+}
+class Program
+{
+    static void Main()
+    {
+        Console.WriteLine(ApplicationMessage.welcomeMessage);
+        while (true)
+        {
+            MenuOption option = GetMenuOption();
+            int num1 = GetIntegerInput("Enter the first number: ");
```

Comment:
Move these messages to static class as well.

Reply:
Done